README.md

```diff
@@ -421,41 +437,11 @@ Once installed, execute `py.test -vv` in the test directory to run all tests.
 
 ### Mac OSX (Limited support for developers)
 
-We've added an installation file to make it easier to run tests and develop on
```

Comment:
This part of the readme needs to be updated to reflect the Conda-based installation.